### ISSUE TITLE
False positive: Class needs to implement __isset to properly work here.

### ISSUE THREAD

**Issue by hackel** [kalessil/phpinspectionsea#897]
In the following simple code example, I'm seeing two issues:
```php
$params = new \Illuminate\Support\Fluent($params);
if (isset($params->value)) {
    //
}
```
Line 1 _warning_: `New value type (\Illuminate\Support\Fluent) is not in annotated types.`
What does this mean?  Is there a way to add it?  I tried adding a class phpDoc block but it had no effect.

Line 2 **error**: `\Illuminate\Support\Fluent needs to implement __isset to properly work here.`
You can see [here](https://github.com/laravel/framework/blob/v5.5.40/src/Illuminate/Support/Fluent.php) that it *does* implement __isset.  Is this a bug, or is there something Laravel needs to fix in its phpDoc?

**Comment by kalessil** [comment]
Hi Ryan,

Line 1: Happens when $params is the a function/method parameter and it's type doesn't include the reported class. In simplest case it's enough to introduce a new local variable instead of re-using the parameter.

Line 2: Might be connected with the previous one, can you check please if tweaking to a new local variable solves this issue as well?

**Comment by kalessil** [comment]
ping @hackel

**Comment by kalessil** [comment]
Closing ticket: can not reproduce
